process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,103 +2,214 @@
 title: DocuSign connector
 ---
 
-The DocuSign connector is used to send documents via email to be digitally signed. The process flow waits for a document to be signed before continuing with the process. The signed document is saved to the Content Services repository. Multiple signers is supported, which means more than one name, email address and signature, can be used.
+The DocuSign connector provides a standard mechanism in Alfresco Process Automation to send documents for digital signing in
+[DocuSign](https://www.docusign.com){:target="_blank"}.
 
-> **Important**: The DocuSign connector requires a [DocuSign](https://www.docusign.com/){:target="_blank"} account to handle document signing. This account is separate to the Alfresco hosted environment and should be created and managed by customers.
+## Installation
 
-The DocuSign connector is displayed on the process diagram as a pen.
```

Comment:
What do you think if we include a section similar to https://docs.alfresco.com/process-automation/latest/model/connectors/content/#create-a-content-service-task? 

The goal of that is explaining how create a docusign task:
![Screenshot 2022-04-12 at 16 31 12](https://user-images.githubusercontent.com/16227735/162986219-006e6aaa-17bb-4e75-8fcf-da250e008dbe.png)

Reply:
I am on vacation from this afternoon I will add this to the JIRA to do it when I am back.

Reply:
Ok, if you can link the JIRA ticket here. Thanks!